CHANGELOG.md

```diff
@@ -33,21 +33,25 @@ All notable changes to this project will be documented in this file.
     * #### Removed
 * ### `nidcpower` (NI-DCPower)
     * #### Added
+        * `get_channel_names` - [#1386](https://github.com/ni/nimi-python/issues/1386) 
```

Comment:
The referenced issue is for nidigital. Can you please split the nidcpower issue out of #1403 into 2 new issues - one for this PR and one for deprecating `get_channel_name`.

Reply:
Created issues [1586](https://github.com/ni/nimi-python/issues/1586) and [1588](https://github.com/ni/nimi-python/issues/1588). I didn't have permissions to modify 1403, but you mentioned you were going to update that one.